src/main/java/teammates/common/util/Config.java

```diff
@@ -95,7 +95,11 @@ private String getGcsBucketname() {
     }
 
     private String getAppUrl() {
-        return props.getProperty("app.url");
+        return Url.trimTrailingSlash(props.getProperty("app.url"));
+    }
+    
+    public static AppUrl getAppUrl(String url) {
+        return new AppUrl(APP_URL, url);
     }
```

Comment:
1. This method should not take any parameters. It should simply return the `app.url` as an `AppUrl` object. Adding relative path onto the base URL should be a method in the Url class, not this class.
2. Add header comments